### PR TITLE
Add minishift-build to minishift namespace

### PR DESCRIPTION
**Pull request by gbraad**

### CODE REVIEW

File: index.d/minishift.yaml [hunk 1]
```diff
@@ -11,3 +11,17 @@ Projects:
     build-context: ./
     depends-on:
       - centos/centos:7
+
+  - id: 2
+    app-id: minishift
+    job-id: minishift-build
```
Comment (LalatenduMohanty) [inline index.d/minishift.yaml]: @gbraad Can we caller it `minishift-builder` image. IMO it sounds better that way.

Comment (gbraad) [reply index.d/minishift.yaml]: it is not really a 'builder'... as it is merely supports the process.

Comment (LalatenduMohanty) [reply index.d/minishift.yaml]: @gbraad Right, it is going help build of Minishift using this image as it contains golang run time. So I still think `builder` is a better name than `build`.

**Comment by praveenkumar** [comment]
@mohammedzee1000 @bamachrn @navidshaikh @dharmit Can anyone of you merge this, one of the PR depend on this.

**Comment by bamachrn** [comment]
@praveenkumar merging it, I believe the renaming discussion is closed off.

**Pull request merged**